apps/mitt-konto/resources/mitt-konto.less

```diff
@@ -201,6 +201,15 @@ h2.mitt-konto---heading {
     }
 }
 
+.mitt-konto--error-message {
+    padding: 0.8em;
+    font-weight: 300;
+    font-size: 1.3em;
+    border-radius: 0 0 0.5em 0.5em;
+    color: @grey-deepdark;
+    background-color: salmon;
+}
+
```

Comment:
We should use stylings from [our styleguide](https://hbl.stage.ksf.infomaker.io/styleguide-2/), e.g. in this case we should use an error notification box:
<img width="535" alt="image" src="https://user-images.githubusercontent.com/5480361/48402657-76a67d80-e72c-11e8-8950-d62c5f8f3458.png">